src/Workspaces/CSharp/Portable/Utilities/CSharpTypeStyleHelper.State.cs

```diff
@@ -14,60 +12,59 @@ namespace Microsoft.CodeAnalysis.CSharp.Utilities
 {
     internal partial class CSharpTypeStyleHelper
     {
-        protected class State
+        protected struct State
```

Comment:
❔ Can we make this `readonly`?

Reply:
most likely.  Let me try that out.

Reply:
Ok.  I've made this readonly.  Will ping you once ci completes.  Thanks!